frontend/js/behaviors/myMuseumTourBuilder/myMuseumTourBuilder.js

```diff
@@ -5,9 +5,11 @@ import * as Sentry from "@sentry/react";
 
 export default function myMuseumTourBuilder(container) {
   this.init = function () {
-    let dataString = container.getAttribute('data-hide-from-tours');
+    let dataStringObjects = container.getAttribute('data-hide-objects-from-tours');
+    let dataStringGalleries = container.getAttribute('data-hide-galleries-from-tours');
     let sentryDsn = container.getAttribute('data-dsn');
-    const hideFromTours = dataString.length > 0 ? dataString.split(",") : [];
+    const hideObjectsFromTours = dataStringObjects.length > 0 ? dataStringObjects.split(",") : [];
+    const hideGalleriesFromTours = dataStringGalleries.length > 0 ? dataStringGalleries.split(",") : [];
```

Comment:
I think this could be a little easier to understand if we used JS's builtin `dataset` functionality and JSON data instead of CSV data and string manipulation:

HTML:
```html
<div data-hide-objects-from-tours="[12345, 67890]">
  ...
</div>
```
Javascript:
```javascript
let hideObjectsFromTours = JSON.parse(container.dataset.hideObjectsFromTours);
```

Reply:
God suggested. I made the changes.